vision/CHANGELOG.md

```diff
@@ -4,6 +4,16 @@
 
 [1]: https://pypi.org/project/google-cloud-vision/#history
 
+## 0.35.1
+
+11-21-2018 11:25 PST
+
+### Implementation Changes
+- Add ProductSearchClient to vision.py ([#6635](https://github.com/googleapis/google-cloud-python/pull/6635))
+
+### Internal / Testing Changes
+- Vision: pick up fixes to GAPIC generator. ([#6632](https://github.com/googleapis/google-cloud-python/pull/6632))
```

Comment:
```suggestion
- Pick up fixes to GAPIC generator. ([#6632](https://github.com/googleapis/google-cloud-python/pull/6632))
```

Reply:
We were frozen and so I was holding this :)